saga-sdk-java/src/main/java/games/mythical/saga/sdk/client/model/SagaBalance.java

```diff
@@ -14,9 +12,4 @@
 public class SagaBalance {
     private String address;
     private String balanceInWei;
```

Comment:
I thought the goal is to remove everything balance related? So I can see not needing address or balance anymore

Reply:
Get item type and item types use the same thing.  Where would i remove it from?

Reply:
nvm I didn't see the changes on currency side, so looks like these fields we'll still support on single GET calls, just multiple will return empty